README.md

```diff
@@ -2,24 +2,22 @@
 
```

Comment:
The README explanation needed to be updated, as well.  I took the chance to also enhance the wording, if possible.

Any wishes for further changes?  :wrench:

Reply:
I think it looks good! 👍